ws-nextjs-app/cypress/e2e/languagesPage/tests.ts

```diff
@@ -0,0 +1,100 @@
+export const homePageTestSuites = [
+  {
+    path: '/ws/languages',
+    service: 'pidgin',
+    runforEnv: ['test'],
+    tests: [
+      ({ service, pageType }: { service: string; pageType: string }) => {
+        describe(`Tests for ${pageType} page - ${service}`, () => {
+          it('should show Test Pidgin Homepage when no renderer_env is set', () => {
+            cy.request({
+              url: 'https://www.test.bbc.com/pidgin',
+              headers: {
+                Origin: 'https://www.test.bbc.com',
+              },
+            }).then(response => {
+              const testContent = response.body;
+
+              cy.get('nav').should('exist');
+              cy.get('main').should('exist');
+
+              cy.get('main').then($main => {
+                const localContent = $main.html();
```

Comment:
Let's remove the word `local` as it can be confused with local environment
```suggestion
                const content = $main.html();
```